tests/pos/irrefutable.scala

```diff
@@ -1,22 +1,11 @@
-// The test which this should perform but does not
-// is that f1 is recognized as irrefutable and f2 is not
-// This can be recognized via the generated classes:
-//
-// A$$anonfun$f1$1.class
-// A$$anonfun$f2$1.class
-// A$$anonfun$f2$2.class
-//
-// The extra one in $f2$ is the filter.
-//
-// !!! Marking with exclamation points so maybe someday
-// this test will be finished.
+// This tests that f1 is recognized as an irrefutable pattern and f2 is not
```

Comment:
Does it really? Will the test fail if `f2` gets suddenly considered as irrefutable (erroneously)?

Reply:
I guess it should be converted to a neg test

Reply:
I replaced this with both a run test and a neg test